### PR TITLE
[NFC][X86] Clang-format X86BaseInfo.h and refine some comments

### PR DESCRIPTION
**Pull request by KanRobert**

### CODE REVIEW

File: llvm/lib/Target/X86/MCTargetDesc/X86BaseInfo.h [hunk 1]
```diff
@@ -22,1299 +22,1286 @@
...
+  /// stub symbols on windows.
+  MO_COFFSTUB,
+};
+
+enum : uint64_t {
+  //===------------------------------------------------------------------===//
+  // Instruction encodings.  These are the standard/most common forms for X86
+  // instructions.
+  //
+  /// PseudoFrm - This represents an instruction that is a pseudo instruction
```
Comment (topperc) [inline llvm/lib/Target/X86/MCTargetDesc/X86BaseInfo.h:510]: The loss of blank lines in this seems worse in some places

Comment (KanRobert) [reply llvm/lib/Target/X86/MCTargetDesc/X86BaseInfo.h:510]: Do you mean the readability?

Comment (topperc) [reply llvm/lib/Target/X86/MCTargetDesc/X86BaseInfo.h:510]: Yes

Comment (KanRobert) [reply llvm/lib/Target/X86/MCTargetDesc/X86BaseInfo.h:510]: We can add some blank lines back if it helps.

**Pull request merged**